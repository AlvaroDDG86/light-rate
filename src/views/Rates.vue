<template>
  <div class="rates">
    <h1>Rates here {{$route.params.day}}</h1>
  </div>
</template>
