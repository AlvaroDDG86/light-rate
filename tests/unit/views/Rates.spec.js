import { shallowMount } from '@vue/test-utils'
import Rates from '@/views/Rates'

describe('rates', () => {
    let wrapper
    beforeEach(() => {
        wrapper = shallowMount(Rates)
    })
    test('should match with snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot()
    });
});