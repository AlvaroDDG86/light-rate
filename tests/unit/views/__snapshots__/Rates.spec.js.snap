// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rates should match with snapshot 1`] = `
<div class="rates bg-gray-50 dark:bg-gray-700">
  <div class="sticky top-0 z-50">
    <banner-stub message=""></banner-stub>
    <div class="items-center flex justify-center bg-gradient-to-r from-yellow-400 via-red-500 to-pink-500 p-2">
      <div class="vue-datepicker text-gray-50">
        <vue-date-picker-stub locale="[object Object]" mobilebreakpoint="576" name="datepicker" validate="true" buttonvalidate="MOSTRAR PRECIOS" buttoncancel="CERRAR" type="date" rangeinputtext="%d ~ %d" value="Mon Feb 14 2022 19:05:16 GMT+0100 (hora estándar de Europa central)" format="DD/MM/YYYY" placeholder="YYYY-MM-DD" color="#ec4899" contentclass="" maxdate="Mon Feb 14 2022 19:05:16 GMT+0100 (hora estándar de Europa central)" visibleyearsnumber="10" nocalendaricon="true" fullscreenmobile="true" tabindex="0" allowoverflow="true" zindex="1002"></vue-date-picker-stub>
      </div>
    </div>
    <h1 class="shadow-md bg-gray-50 dark:bg-gray-700 text-2xl font-light px-10 text-center dark:text-gray-50"></h1>
  </div>
  <div class="flex justify-around flex-wrap gap-y-0.5 gap-x-0.5"></div>
</div>
`;
